src/V3UniqueNames.h

```diff
@@ -28,14 +28,24 @@
 #include <unordered_map>
 
 class V3UniqueNames final {
+public:
+    enum runMode {
+        NORMAL,
+        NO_SUFFIX  // V3UniqueNames will not add a suffix to the name if it is unique.
+    };
+
+private:
     const std::string m_prefix;  // Prefix to attach to all names
 
     std::unordered_map<std::string, unsigned> m_multiplicity;  // Suffix number for given key
 
+    const runMode mode = NORMAL;
```

Comment:
```suggestion
    const runMode m_mode = NORMAL;  // Ad suffix or not
```

Reply:
Fixed

Reply:
Note you can hit "Resolve conversation" in github.com rather than replying fixed, so that way the item gets hidden in the issue comments.